### ISSUE TITLE
The Win32DesktopApp XamlIsland sample is failing Accessibility Insights test

### ISSUE THREAD

**Issue by wayvad** [marb2000/XamlIslands#27]
The Win32DesktopApp sample is great at getting us started with a Xaml app, but our app is failing the accessibility test (via the Accessibility Insights tool). I tested the sample (unmodified), and it has the same accessibility issue as well:

![image](https://user-images.githubusercontent.com/40037392/110683319-b660c100-8190-11eb-96ad-c25fe779ecbd.png)

Is this a known issue? Thanks.
